### PR TITLE
libvori: init at 210412

### PR DESCRIPTION
**Pull request by sheepforce**
###### Motivation for this change
Adds libvori, which is a missing dependency for CP2K (quantum chemistry, molecular dynamics, ...). Part of the packages @markuskowa and me want to merge from https://github.com/markuskowa/NixOS-QChem/issues/56 .

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/libvori/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchurl, cmake } :
+
+stdenv.mkDerivation rec {
+  pname = "libvori";
+  version = "210412";
+
+  nativeBuildInputs = [ cmake ];
+
+  # Original server is misconfigured and messes up the file compression.
```
Comment (markuskowa) [inline pkgs/development/libraries/libvori/default.nix:9]: The original download location seems to work for me: `https://brehm-research.de/files/libvori-210412.tar.gz`

Comment (sheepforce) [reply pkgs/development/libraries/libvori/default.nix]: Nice, so this has been fixed recently. Changed it.

File: pkgs/development/libraries/libvori/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, cmake } :
+
+stdenv.mkDerivation rec {
+  pname = "libvori";
+  version = "210412";
+
+  nativeBuildInputs = [ cmake ];
+
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libvori/default.nix:8]:
```suggestion
```

File: pkgs/development/libraries/libvori/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  src = fetchurl {
+    url = "https://brehm-research.de/files/${pname}-${version}.tar.gz";
+    sha256 = "1b4hpwibf3k7gl6n984l3wdi0zyl2fmpz84m9g2di4yhm6p8c61k";
+  };
+
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libvori/default.nix:13]:
```suggestion

  nativeBuildInputs = [ cmake ];

```

File: pkgs/development/libraries/libvori/default.nix [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{ stdenv, lib, fetchurl, cmake } :
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libvori/default.nix:1]:
```suggestion
{ stdenv, lib, fetchurl, cmake }:
```

**Pull request merged**